src_c/color.c

```diff
@@ -1841,10 +1841,10 @@ _color_slice(register pgColorObject *a, register Py_ssize_t ilow,
         return Py_BuildValue("(iiii)", c1, c2, c3, c4);
     }
     else if (len == 3) {
-        return Py_BuildValue("(iii)", c1, c2, c3);
+        return pg_tuple_triple_from_values_int(c1, c2, c3);
     }
     else if (len == 2) {
-        return Py_BuildValue("(ii)", c1, c2);
+        return pg_tuple_couple_from_values_int(c1, c2);
```

Comment:
These should be cast to ints

Reply:
Yes-- this is why the appveyor builds are failing.